plugins/MUN Identity/__init__.py

```diff
@@ -139,4 +162,4 @@ async def identity_associate(
             self.bot.get_guild(PRIMARY_GUILD).get_role(VERIFIED_ROLE),
             reason=f"Identity manually associated by {ctx.author.name}#{ctx.author.discriminator}. MUN username: {mun_username}",
         )
-        await ctx.send("Identity associated.")
+        await ctx.send("Identity associated.")
```

Comment:
```suggestion
        await ctx.send("Identity associated.")

```
This newline shouldn't be removed